missions/2PzD_Template_v3_0.VR/customization/geardefs/Russian/gearDefRusRegular.sqf

```diff
@@ -1,21 +1,38 @@
 //======================== Russian Regular Uniform Definitions ========================
 
-//Uniform
-#define Rus_Uni_BC                              "U_LIB_SOV_Kapitan"
-#define Rus_Uni_CC                              "U_LIB_SOV_Stleutenant"
-#define Rus_Uni_CSgt                            "U_LIB_SOV_Starshina"
-#define Rus_Uni_PC                              ["U_LIB_SOV_Leutenant"],["U_LIB_SOV_Leutenant_inset_pocket"]
-#define Rus_Uni_PSgt                            "U_LIB_SOV_Stsergeant"
-#define Rus_Uni_SL                              "U_LIB_SOV_Sergeant_inset_pocket"
-#define Rus_Uni_TL                              "U_LIB_SOV_Efreitor"
-#define Rus_Uni_Rif                             "U_LIB_SOV_Strelok"
+//1941 Uniforms
+#define Rus_Uni41_BC                            "U_NORTH_SOV_Obr35_Uniform_Cpt"
+#define Rus_Uni41_CC                            "U_NORTH_SOV_Obr35_Uniform_1stLt"
+#define Rus_Uni41_CSgt                          "U_NORTH_SOV_Obr35_Uniform_Lt"
```

Comment:
I'd give this one a Sergeant uniform as well, instead of a Lieutenant one. Perhaps also change the Company Commander to have a Captain's uniform since that rank fits the position better. We can then simply remove the Battalion Commander (unless we actually have a Major uniform in any of the mods?) since we don't use it anyway. Not that we use the CC very often (or at all) but at least we have the appropriate uniform for that one.